commands/command_push.go

```diff
@@ -31,28 +32,28 @@ var (
 // of commits between the local and remote git servers.
 func pushCommand(cmd *cobra.Command, args []string) {
 	if len(args) == 0 {
-		Print("Specify a remote and a remote branch name (`git lfs push origin main`)")
+		Print(tr.Tr.Get("Specify a remote and a remote branch name (`git lfs push origin main`)"))
 		os.Exit(1)
 	}
 
 	requireGitVersion()
 
 	// Remote is first arg
 	if err := cfg.SetValidPushRemote(args[0]); err != nil {
-		Exit("Invalid remote name %q: %s", args[0], err)
+		Exit(tr.Tr.Get("Invalid remote name %q: %s", args[0], err))
 	}
 
 	ctx := newUploadContext(pushDryRun)
 	if pushObjectIDs {
 		if len(args) < 2 {
-			Print("Usage: git lfs push --object-id <remote> <lfs-object-id> [lfs-object-id] ...")
+			Print(tr.Tr.Get("At least one object ID must be supplied with --object-id"))
 			return
 		}
 
 		uploadsWithObjectIDs(ctx, args[1:])
 	} else {
 		if len(args) < 1 {
-			Print("Usage: git lfs push --dry-run <remote> [ref]")
+			Print(tr.Tr.Get("A remote is required"))
 			return
 		}
```

Comment:
Out of interest, is it possible to trigger this message?  From a cursory glance it seems like, if `len(args) < 1`, then it must be zero, which would be caught above where `Specify a remote and a remote branch name` is output.  Maybe this block could be removed?